P5/Source/Guidelines/en/PH-PrimarySources.xml

```diff
@@ -2664,6 +2664,21 @@ referring to the zone marked in purple on the right
             annotation. </p>
       </div>
    </div>
+   
+   
+   <div  type="div2" xml:id="PHRB">
+      <head>Transcription and Ruby</head>
+      
+      <p>These Guidelines also provide special elements to support the encoding of
+         Ruby annotations, which are common in East Asian textual traditions. These
```

Comment:
In section 3.4.2 "ruby" is written in lower case (in running text).

Reply:
Done! Thanks Martina.